cli/client.go

```diff
@@ -1044,6 +1045,10 @@ var clientRetrieveCmd = &cli.Command{
 			Name:  "miner",
 			Usage: "miner address for retrieval, if not present it'll use local discovery",
 		},
+		&cli.StringFlag{
+			Name:  "datamodel-path-selector",
+			Usage: "a rudimentary (DM-level-only) text-path selector, allowing for sub-selection within a deal",
```

Comment:
do we imagine extending this to ADL-extended selectors soon?
would that be a different flag?
it seems fine for this to just be a `path-selector` rather than including all the `datamodel-only` warnings on the interface.

Reply:
Right... this goes back to the mega-thread, where @aschmahmann and me were pushing against transparent-unixfs in the low-level selection.
I personally would not use a heuristic-engaging selector on principle, and left the door open for "do not try to be clever here" with the `datamodel` prefix. I am open to switching this out, but we would need signaling "I want to use this as a low level selector" in some fashion.